### DISCUSSION TITLE
¿Es normal que se tarde ejcutando codigo?

### DISCUSSION THREAD

**Discussion by jaaramirez** [LaunchX-InnovaccionVirtual/CursoIntroPython#244] [Q&A]
Hola, tengo una duda estoy en la kata 5 en el ejercicio dos cuando ejecuto codigo se tarda y no me arroja resultado lo maximo que lo deje correr fueron 20 min y lo interrumpi, lo borre y lo intetnte con el codigo de solución que brindan al final y me sigue pasando lo mismo no se si puse algo mal o tengo mal los datos ya corregi y me sigue taradando en ejecutar el codigo alguien podría orientarme, gracias, buena tarde 

[
![image](https://user-images.githubusercontent.com/89113801/153955305-f2d925e7-f35d-4412-a966-739f882589b0.png)
](url)

**Comment by OscarPedroza** [comment]
Hola, en la parte superior te aparece un input para que introducir la información, por eso marca ese tiempo, porque se queda a la espera.

Sería este input:

![image](https://user-images.githubusercontent.com/83786890/153955687-98e40f18-a26c-48f0-8ab9-753d5747ed21.png)

**Reply by doguedogue** [reply]
Con el vs code en modo dark a veces no se ve el input, se camuflajea con el IDE.

Llena el valor del input y enter

**Comment by ArielCanedo** [comment]
hola, pusiste la distancia que te pide en el recuadro de arriba en el
centro?

**Reply by jaaramirez** [reply]
Muchas gracias por su apoyo, era correcto me faltaba meter las distancias, gracias
